koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -2273,3 +2274,22 @@ def delete_infrastructure_raw_cost_from_daily_summary(self, provider_uuid, repor
         """
 
         self._execute_raw_sql_query(table_name, sql, start_date, end_date)
+
+    def _execute_processing_script(self, script_file_path, sql_params):
```

Comment:
My concern here is this introduces two different ways of running a SQL file, even within the same ocp_report_db_accessor. Through the rest of the code base we do something like: https://github.com/project-koku/koku/blob/main/koku/masu/database/ocp_report_db_accessor.py#L563-L576

What's the perceived benefit of this?

Reply:
@adberglund 
The benefit, as I see it is being able to track the statements atomically if there is an error or inefficiency. Instead of sending up a script to be executed, when sending statements one-at-a-time, I can see them discretely in the logs and see them easier in the AWS insights panel.

Reply:
Directive will be to take the split logic and apply it to the existing function, then utilize that function instead of `_execute_processing_script` new func.

Reply:
Discussed this with Andrew and we have agreed upon deferring this work to another ticket for tech-debt cleanup. It was leading to numerous test failures retrofitting the SQL split into the existing code with the existing scripts.